openlibrary/plugins/upstream/utils.py

```diff
@@ -212,6 +212,38 @@ def render_component(
     return html
 
 
+def render_macro(name, args, **kwargs):
+    return dict(web.template.Template.globals['macros'][name](*args, **kwargs))
```

Comment:
I wonder if we should be using Infogami's [`call_macro` function](https://github.com/internetarchive/infogami/blob/master/infogami/utils/macro.py#L100-L116) here?